extensions/pkg/webhook/controlplane/controlplane.go

```diff
@@ -83,7 +83,7 @@ func New(mgr manager.Manager, args Args) (*extensionswebhook.Webhook, error) {
 func getName(kind string) string {
 	switch kind {
 	case KindSeed:
-		return ExposureWebhookName
+		return SeedProviderWebhookName
```

Comment:
Hm, isn't there action required for extension developers? I would expect to see a release note of type breaking for extension developers with instructions what they need to change.

Reply:
I rephrased the release notes and split them up. So I now suggest to have the breaking naming change as well as the hint to change the package name in the extension.